app/mainwindow.ui

```diff
@@ -816,6 +817,17 @@
     <string>&amp;Diagram Support</string>
    </property>
   </action>
+  <action name="actionSourceAtOneSize">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Source at one size </string>
```

Comment:
I'm not a native speaker, but doesn't "Source at Single Size" sound better? Also please use title capitalization for this (See here for a description: https://techbase.kde.org/Projects/Usability/HIG/Capitalization).